app/helpers/application_form_builder.rb

```diff
@@ -8,15 +8,17 @@ def initialize(object_name, object, template, options)
 
   (field_helpers - [ :label, :check_box, :radio_button, :fields_for, :fields, :hidden_field, :file_field ]).each do |selector|
     class_eval <<-RUBY_EVAL, __FILE__, __LINE__ + 1
-      def #{selector}(method, options)
+      def #{selector}(method, options, &block)
         default_options = { class: "form-field__input" }
         merged_options = default_options.merge(options)
+        content = block_given? ? @template.capture(&block) : ''
```

Comment:
This feels a little off to me, it may be better to just use the helpers directly for a complex field like the currency dropdown. Sometimes a little repetition is okay.

```
<% form_field_tag do %>
  <% label :balance, class: "form-field__label" %>
  <%# inputs here %>
<% end %>
```

This would give you more control over the styling and placement of the contents of the field. The designs indicate at some point we might want to put the currency symbol at the beginning for example.

This would also enable you to wrap the symbol and inputs with a container and style them with flexbox then.